mmv1/products/networkservices/EdgeCacheOrigin.yaml

```diff
@@ -332,3 +332,26 @@ properties:
         item_type:
           type: String
         max_size: 5
+  - name: 'flexShielding'
+    type: NestedObject
+    description: |
+      The FlexShieldingOptions to be used for all routes to this origin.
+
+      If not set, defaults to a global caching layer in front of the origin.
+    properties:
+      - name: flexShieldingRegions
```

Comment:
Can this field be updated? If so can we add an update test?

Reply:
Sorry, updated as in can we add new regions in the future? Yes but I'm not sure how to test that. Or updated as in users can update the value? I can set it to a different value in the EdgeCacheOrigin_update_1 test if that's sufficient

Reply:
Sorry, updated as in can we add new regions in the future?
- Are you expecting the region list to be highly volatile? If so I would recommend using string for this with a external link to valid values in the description. (markdown is supported). This is recommended as maintenance can become a burden. 

Or updated as in users can update the value? I can set it to a different value in the EdgeCacheOrigin_update_1 test if that's sufficient.
- Yep that would work !